doc/src/manual/strings.md

```diff
@@ -1138,7 +1138,7 @@ order to also allow for unstable (i.e. pre-release) `0.2` versions, the lower bo
 be modified like this: `v"0.2-" <= VERSION`.
 
 Another non-standard version specification extension allows one to use a trailing `+` to express
-an upper limit on build versions, e.g.  `VERSION > v"0.2-rc1+"` can be used to mean any version
+a lower limit on build versions, e.g.  `VERSION > v"0.2-rc1+"` can be used to mean any version
```

Comment:
This change is incorrect, though the original may be inartfully worded. In particular what this is meant to say with that `v"0.2-rc1+" >= v"0.2-rc1+FOO"` for any value of FOO.

Reply:
I see. Thank you @Keno for clarifying.

Reply:
Well, if you want to have a go at it, this could still use clarifying probably, but it is technically correct as written, even if confusing.